engine/launcher/src/main/scala/org/enso/launcher/cli/LauncherLogging.scala

```diff
@@ -35,10 +30,9 @@ object LauncherLogging extends LoggingServiceSetupHelper {
     * This is necessary on Windows to ensure that the logs file is closed, so
     * that the log directory can be removed.
     */
-  def prepareForUninstall(colorMode: ColorMode): Unit = {
+  def prepareForUninstall(logLevel: Option[Level]): Unit = {
     waitForSetup()
-    LoggingServiceManager.replaceWithFallback(printers =
-      Seq(stderrPrinter(colorMode, printExceptions = true))
-    )
+    val actualLogLevel = logLevel.getOrElse(defaultLogLevel)
+    LogbackSetup.get().setupConsoleAppender(actualLogLevel)
```

Comment:
No (compile time) dependency on `LogbackSetup` please. Just use the [LoggerSetup.get static method](https://github.com/enso-org/enso/pull/7559/files#r1303229897) here.